### DISCUSSION TITLE
how to work the script ?

### DISCUSSION THREAD

**Discussion by cisko99za** [SpotX-Official/SpotX#663] [Question]
SpotX it's a wonderful project and i'm impressed about how the script works. but i'm curious. after run the script all spotify files remain the same (in size), and except spotify.bak (the old spotify.bak) can't see difference between files. also renamed the new spotify.exe with another name and spotify.bak as spotify.exe (the original exe file) the app is still patched
so, how to work this script ? to whole system or patch files only ? i think files only, because copy spotify folder from user folder to another partition, then uninstall spotify from my pc, on new run the app is still patched, but the files look always the same ...

**Comment by amd64fox** [comment]
spotx mainly patches this file:
`\Roaming\Spotify\Apps\xpui.spa`

and to a lesser extent this binary file:
`\Roaming\Spotify\Spotify.exe`

In the case of the binary file, we need to maintain its size exactly down to the byte; otherwise, it will simply be broken.

**Comment by cisko99za** [comment]
thanks for explanation
